README.md

```diff
@@ -125,20 +125,25 @@ $ kubectl -n <namespace> apply -f csi.ibm.com_v1_ibmblockcsi_cr.yaml
 
 ```bash
 $ kubectl get all -n <namespace>  -l csi
-NAME                             READY   STATUS    RESTARTS   AGE
-pod/ibm-block-csi-controller-0   6/6     Running   0          9m36s
-pod/ibm-block-csi-node-jvmvh     3/3     Running   0          9m36s
-pod/ibm-block-csi-node-tsppw     3/3     Running   0          9m36s
+NAME                                            READY   STATUS    RESTARTS   AGE
+pod/ibm-block-csi-controller-0                  6/6     Running   0          9m36s
+pod/ibm-block-csi-node-jvmvh                    3/3     Running   0          9m36s
+pod/ibm-block-csi-node-tsppw                    3/3     Running   0          9m36s
+pod/ibm-block-csi-operator-5bb7996b86-xntss     1/1     Running   0          10m
 
 NAME                                DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
 daemonset.apps/ibm-block-csi-node   2         2         2       2            2           <none>          9m36s
 
+NAME                                     READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/ibm-block-csi-operator   1/1     1            1           10m
+
+NAME                                                DESIRED   CURRENT   READY                                                                                                                                 AGE
```

Comment:
isn't the `AGE` here a bit over-indented? where's it from?

Reply:
oh wow, I didn't see that! done